ui/plugin-system/src/stories/shared-utils/decorators/WithPluginRegistry.tsx

```diff
@@ -22,9 +22,9 @@ import {
 // NOTE: the aliases we use for components break these top level imports, so we
 // import relatively.
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const prometheusResource = require('../../../../prometheus-plugin/plugin.json');
+const prometheusResource = require('../../../../../prometheus-plugin/plugin.json');
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const panelsResource = require('../../../../panels-plugin/plugin.json');
+const panelsResource = require('../../../../../panels-plugin/plugin.json');
```

Comment:
Should the stories that need metadata from panels-plugin live in a higher level package like dashboards, is the long relative url to get around creating a circular dependency? 

I guess maybe this is indicative of the “package reorg” situation that I hope we can get prioritized at some point

Reply:
I think I should still do ^, but I'm going to put it in a separate PR because it's a big enough refactor I'd like another round of review, and it'd be nice to get this first change that folks are happy with out without blocking that.